material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -85,7 +87,11 @@
   --md-mermaid-sequence-note-border-color: var(--md-mermaid-label-fg-color);
   --md-mermaid-sequence-number-bg-color: var(--md-mermaid-node-fg-color);
   --md-mermaid-sequence-number-fg-color: var(--md-accent-bg-color);
-}
+
+  /*custom code admonition colors*/
+
+  --md-code-admo-border-color: var(--code-green);
+  --md-code-admo-title-background: var(--code-green-light);
```

Comment:
This is part of the `:root,
[data-md-color-scheme='default'] {` definintion. So it needs to be enclosed as such. That is why there is a white trim showing up on the left and right sides of the page